### ISSUE TITLE
[Adaptive Card iOS - Flight Itinerary]: Voice Over is conveying irrelevant information as double tap to edit for the 'Your Flight Itinerary'' heading.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards-Mobile#25]
**Test Environment:**
Device: iPhone 11
iOS Version: 16.7.1
App Version: 1.0 (2.4.0-beta.23.10.15.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.5" control and select it.
5. Navigate to 'Scenario' control and activate it.
6. Navigate to 'FlightItinerary.JSONS' and double tap to activate it.
7. The Card will open.
8. Perform right swipe, navigate to 'Your Flight Itinerary' heading and observe the issue.

**Actual Result:**
While navigating the 'Flight Itinerary' card, Voice Over is conveying the irrelevant information as 'Double tap to edit' for the non-interactive heading. It is announcing as **'Your Flight Itinerary heading Your Flight Itinerary heading text field group double tap to edit.'**.

**Expected Result:**
Voice Over should not convey the irrelevant information for the heading and when focus lands on it, VO should announce as **'Your Flight Itinerary heading'.**

**User Impact:**
It will impact the screen reader user as they will try to perform action on the non interactive element if the screen reader announce the information as Double tap to activate it.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/f9b7ecec-13f3-4af0-940d-dc47a348bd14

**Comment by vagpt** [comment]
Closing this issue as this is tracking with #24
